src/OrleansPSUtils/OrleansPSUtils.2017.csproj

```diff
@@ -0,0 +1,59 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup Label="NuGet">
+    <IsPackable>true</IsPackable>
+    <PackageId>Microsoft.Orleans.OrleansPSUtils</PackageId>
```

Comment:
I don't remember now why I used that 3rd party library. I think it was suggested by someone from PS team by that time because of some limitation on the official nuget. However, I've already spoke with the PS team again and I'll port that extension to use the new xplat powershell nugets.